packages/vue3-vite/src/generators/library/files/vite.config.ts.template

```diff
@@ -39,8 +41,9 @@ module.exports = defineConfig({
         globals: {
           vue: 'Vue'
         }
-      }
-    }
+      },
+      plugins: [copyPackageJsonPlugin(__dirname)],
+    },
```

Comment:
This is how the plugin can be used; I added it to both of the `vite.config.ts` in vue3-vite and library.
It only copies `package.json` to `dist` if it exists.